src/rgw/jwt-cpp/jwt.h

```diff
@@ -11,6 +11,10 @@
 #include <openssl/pem.h>
 #include <openssl/ec.h>
 #include <openssl/err.h>
+#include <openssl/bn.h>
+#include <openssl/rsa.h>
+
+#include "../rgw_b64.h"
```

Comment:
style nit: I don't like ".." in include filenames.  Might be best to call this "rgw-jwt.h" instead of "jwt-cpp/jwt.h".

Reply:
+1